dingo/gurobi_based_implementations.py

```diff
@@ -108,8 +108,8 @@ def fast_fba(lb, ub, S, c):
                     optimum_value = -model.getObjective().getValue()
                     v = model.getVars()
 
-                for i in range(n):
-                    optimum_sol.append(v[i].x)
+                    for i in range(n):
+                        optimum_sol.append(v[i].x)
 
                 optimum_sol = np.asarray(optimum_sol)
```

Comment:
Should this also be in under the if case? Also should we add a case for "status != GRB.OPTIMAL"? What values are returned in that case?

Reply:
Similar to the `slow_fba` function, the `optimum_sol` variable holds significance only in the optimal case. 

If the status is not equal to `GRB.OPTIMAL`, the `fba` function will return the initial values of `optimum_sol` and `optimum_value`, with `optimum_value` defaulting to 0.

Reply:
Yes, so `optimum_sol = np.asarray(optimum_sol)` is not needed to be under if  `status == GRB.OPTIMAL`
I am also thinking why not just writing 
```
optimum_sol = np.asarray(model.getVars())
```

Reply:
`model.getVars()` will return all Gurobi variable objects rather than numeric values.

Reply:
I see, then why not `model.getAttr("X", v)` or something. Of course this is not obligatory for your PR but since you are changing this part of code why not remove that for loop too.

Reply:
OK, I have removed the loop.